docs/explain-provisioning.md

```diff
@@ -0,0 +1,16 @@
+# What is provisioning?
+
+You may have heard or read about the word provisioning inside or outside Zulip.
+This documentation explains what do we mean when we implicitly speak about provisioning.
+
+## Background
+
+Server provisioning is a set of actions to prepare a server with appropriate systems, data and software, and make
+it ready for network operation. Typical tasks when provisioning a server are: select a server from a pool of available servers,
+load the appropriate software (operating system, device drivers, middleware, and applications), appropriately customize and configure
+the system and the software to create or change a boot image for this server, and then change its parameters, such as IP address,
+IP Gateway to find associated network and storage resources (sometimes separated as resource provisioning) to audit the system.
+
+Inside Zulip the first time we want to run our server, we need to do some provisioning for its proper functioning.
+This includes running a series of tests that tunes our system, install appropiate software and populate our database to
```

Comment:
Tests are not part of provisioning. In provisioning we just install the software.